src/components/detail-carousel.jsx

```diff
@@ -248,6 +248,9 @@ const DetailCarousel = ({widget, widgetHeight=''}) => {
 
 	const snapToImage = (fast=false) => {
 		const _pics = picScrollerRef.current
+		if(!_pics){
```

Comment:
Single line expressions in a function can be condensed to:
```
if ( ! _pics) return false
```

If using `return` to short-circuit a function, prefer `return false`.